src/gc/impl/conservative/gc.d

```diff
@@ -3078,7 +3279,7 @@ struct Pool
             bPageOffsets = null;
         }
 
-        mark.Dtor();
+        mark.Dtor(config.fork);
```

Comment:
Nit: this is an unconditional call, while `mark.alloc` uses `version(COLLECT_FORK)`. They should be consistent, probably simpler to leave the evaluation of the  `share` parameter to `GCbits` and remove the version condition for `mark.alloc`..